app/carbonapi/app.go

```diff
@@ -51,43 +52,62 @@ type App struct {
 	defaultTimeZone *time.Location
 
 	backends []backend.Backend
+
+	prometheusMetrics PrometheusMetrics
```

Comment:
It's a very generic suggestion, but I believe it still has a place here: I might suggest isolating Prometheus in a stand-alone module and implementing an adapter layer. The motivation for this change is to avoid the technology lock in the future.

Reply:
Thanks for the suggestion. We will consider doing this.